.github/workflows/add-to-project.yml

```diff
@@ -0,0 +1,18 @@
+name: Add new issues to spark-rapids project
+
+on:
+  issues:
+    types:
+      - opened
+
+jobs:
+  add-to-project:
+    name: Add new issues to spark-rapids project
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/add-to-project@RELEASE_VERSION
```

Comment:
RELEASE_VERSION should be a specific version, https://github.com/actions/add-to-project/releases

Reply:
I updated the RELEASE_VERSION to be the latest (v0.3.0).

However, after setting the default branch, I don't see a way to create issues in my forked repo to test out the Action: https://github.com/mattahrens/spark-rapids.  Where can I create test issues to validate the workflow?

Reply:
I figured out a way to add Issues to my forked repo so I tested it out.  This is the error I'm seeing: 

```
Request failed due to following response errors: - Resource not accessible by integration
```

I'll debug the token I'm using to see if I can work through it.

Reply:
I got the action to succeed by adding a repository secret in the repo Settings -> Secrets -> Action -> New Repository Secret.  For this test, I used my PAT.

Workflow success: https://github.com/mattahrens/spark-rapids/actions/runs/3152929749
Project board update: https://github.com/users/mattahrens/projects/2

Let me know next steps in terms of how to change it for actually merging for Spark RAPIDS.  I can update the project-url to be https://github.com/orgs/NVIDIA/projects/4, but I'm not sure what to use for the github-token in the Spark RAPIDS repo.  Hopefully I can reuse a secret that is already set up as a repository secret.